src/uu/tr/src/operation.rs

```diff
@@ -189,6 +190,9 @@ impl Sequence {
                     },
                 };
                 let mut set1_solved: Vec<_> = set1.iter().flat_map(Self::flatten).collect();
+                if complement_flag {
+                    set1_solved = (0..=u8::MAX).filter(|x| !set1_solved.contains(x)).collect();
+                }
                 if truncate_set1_flag {
                     set1_solved.truncate(set2_solved.len());
                 }
```

Comment:
Shouldn't these to `if`s be flipped around?

```
$ echo asdfqwer | ../gnu/src/tr -dts asdf qwe
qwer
$ echo asdfqwer | cargo run tr -dts asdf qwe
fqwer
```

Todo for the future, out-of-scope for this PR: Measure the performance impact. `set1_solved` can be arbitrarily long, and I think `.contains()` is linear time. If that's the case, then there's probably a faster approach (by factor 255), e.g. allocating a `vec![false; 256]` and marking which bytes are seen in `set1_solved`, and going from there. Again, out-of-scope, I just want to write down this thought somewhere.

Reply:
That should be easy enough to implement right here c:

Reply:
Thinking more about it, I might want to change set2 too (clamp it to 256 elements). So it might be better to split it to another PR.

Reply:
> echo asdfqwer | ../gnu/src/tr -dts asdf qwe

This is another separate issue (the complement is not running here)

Reply:
> Translation  occurs  if  -d  is not given and both STRING1 and STRING2 appear.  -t is only significant when translating.

From the man page, apparently `-t` should be ignored if `-d` is given, or `STRING2` is missing.

This should be easy to add too

Reply:
Ah, I forgot to check if it happens on main. Feel free to handle it in this PR, or maybe a different PR has to tackle it. You decide.